From e17f3b92025a6067874c41c6c7ce3cffedfb6afa Mon Sep 17 00:00:00 2001
From: Frederic Crozat <fcrozat@suse.com>
Date: Thu, 12 Jun 2014 15:01:37 +0200
Subject: [PATCH 102/119] device-manager: only call ply_terminal_free

ply_terminal_free will call ply_terminal_close anyway and is guarded
against NULL terminal (ply_terminal_close is not).
---
 src/libply-splash-core/ply-device-manager.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libply-splash-core/ply-device-manager.c b/src/libply-splash-core/ply-device-manager.c
index e904708..8dce68b 100644
--- a/src/libply-splash-core/ply-device-manager.c
+++ b/src/libply-splash-core/ply-device-manager.c
@@ -412,7 +412,6 @@ free_terminal (char                 *device,
 {
         ply_hashtable_remove (manager->terminals, device);
 
-        ply_terminal_close (terminal);
         ply_terminal_free (terminal);
 }
 
-- 
1.9.0

